api_app/analyzers_manager/file_analyzers/cape_sandbox.py

```diff
@@ -0,0 +1,145 @@
+import time
+
+import requests
+
+from api_app.analyzers_manager.classes import AnalyzerRunException, FileAnalyzer
+from tests.mock_utils import MockResponse, if_mock_connections, patch
+
+
+class CAPEsandbox(FileAnalyzer):
+    max_tries: int = 25
+    to_respond = {}
+
+    def set_params(self, params):
+        self.__token = self._secrets["token"]
+        self.__vm_name = params.get("VM_NAME", "win7x64_8")
+        self.BASE_URL = self._secrets.get("url_key_name", "https://www.capesandbox.com")
+
+    def run(self):
+        API_URL: str = self.BASE_URL + "/apiv2/tasks/create/file/"
+        self.headers = {
```

Comment:
create a `self.session = requests.Session(headers={..})` inside `set_params` method and use this session object for making requests.

Reply:
oh right! This is a better way to proceed forward with multiple requests.